File: JS/URI/1012.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var leia = lines[0];

var medidas = leia.split(' ');



console.log("TRIANGULO:", ((medidas[0]*medidas[2])/2).toFixed(3));
console.log("CIRCULO:", (3.14159*(medidas[2]*medidas[2])).toFixed(3));
console.log("TRAPEZIO:", ((parseFloat(medidas[0])+parseFloat(medidas[1]))*medidas[2]/2).toFixed(3));
console.log("QUADRADO:", (medidas[1]*medidas[1]).toFixed(3));
console.log("RETANGULO:", (medidas[0]*medidas[1]).toFixed(3));

File: Python/URI/1012.py
medidas = input().split(" ");

a, b, c = medidas;

triangulo = float(a)*float(c)/2.0;
circulo = 3.14159*float(c)*float(c);
trapezio = (float(c)*(float(a)+float(b)))/2;
quadrado = float(b)*float(b);
retangulo = float(a)*float(b);

print("TRIANGULO: %0.3f" %triangulo);
print("CIRCULO: %0.3f" %circulo);
print("TRAPEZIO: %0.3f" %trapezio);
print("QUADRADO: %0.3f" %quadrado);
print("RETANGULO: %0.3f" %retangulo);